compiler/src/dotty/tools/dotc/quoted/QuoteDriver.scala

```diff
@@ -13,10 +13,9 @@ import java.nio.charset.StandardCharsets
 
 class QuoteDriver extends Driver {
 
-  def run[T](expr: Expr[T]): T = {
+  def run[T](expr: Expr[T], optimise: Boolean): T = {
```

Comment:
That will work for now, but in the future if you want to be able to pass more options, it might make more sense to pass a `Settings` object.

Reply:
I will add it now to avoid future changes on the API.